protos/core.proto

```diff
@@ -424,4 +429,12 @@ message TableMetadata {
   Target target = 1;
   string type = 2; // "table" or "view"
   repeated Field fields = 3;
+  int64 lastUpdatedMillis = 4;
+}
+
+message PersistedTableMetadata {
+  TableMetadata table = 1;
```

Comment:
I don't think you need any of this in the metadata table? It's information about columns and types which I don't think we need to keep track of.

Reply:
moved to another message